recirq/qaoa/placement.py

```diff
@@ -3,17 +3,25 @@
 from functools import lru_cache
 from typing import Callable, Dict, List, Optional, Tuple
 
+import networkx as nx
+import numpy as np
+
 import cirq
 import cirq.contrib.routing as ccr
 import cirq_google as cg
-import networkx as nx
-import numpy as np
-import pytket
-import pytket.extensions.cirq
-from pytket.circuit import Node, Qubit
-from pytket.passes import SequencePass, RoutingPass, PlacementPass
-from pytket.predicates import CompilationUnit, ConnectivityPredicate
-from pytket.routing import GraphPlacement
+
+try:
```

Comment:
Maybe a quick comment here to explain that this is here for CI tests?

Reply:
added:
```
    # Set the 'RECIRQ_IGNORE_PYTKET' environment variable to treat PyTket as an optional
    # dependency. We do this for CI testing against the next, pre-release Cirq version.
```

(in general, users could also set this environment variable if they get themselves into hot water managing the pytket dependency)